sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/ShowPartitionsSuite.scala

```diff
@@ -117,6 +119,22 @@ class ShowPartitionsSuite extends ShowPartitionsSuiteBase with CommandSuiteBase
     }
   }
 
+  test("show partitions of non-partitioned table") {
+    withNamespaceAndTable("ns", "not_partitioned_table") { t =>
+      sql(s"CREATE TABLE $t (col1 int) $defaultUsing")
+      val sqlText = s"SHOW PARTITIONS $t"
+      val tableName =
+        UnresolvedAttribute.parseAttributeName(t).map(quoteIdentifier).mkString(".")
+
+      checkError(
+        exception = intercept[AnalysisException] {
+          sql(sqlText)
+        },
+        errorClass = "_LEGACY_ERROR_TEMP_1269",
```

Comment:
Should we migrate the error class `_LEGACY_ERROR_TEMP_1269` on to `PARTITION_SCHEMA_IS_EMPTY`?

Reply:
OK